### PR TITLE
test: Rename AKS test cluster

### PR DESCRIPTION
**Pull request by tomkerkhove**
As we are going to a new Azure subscription, now might be the perfect time to rename and align naming conventions.

In this case, I'd like to standardize the AKS cluster name.

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Related to https://github.com/kedacore/testing-infrastructure/issues/107

### CODE REVIEW

**Comment by tomkerkhove** [comment]
I used github.dev again which broke DCO which I'm fixing now

File: terraform/main.tf [hunk 1]
```diff
@@ -57,7 +57,7 @@ module "azure_aks_pr" {
   source              = "./modules/azure/aks"
   resource_group_name = var.azure_resource_group_name
   kubernetes_version  = "1.25"
-  cluster_name        = "keda-pr-run"
+  cluster_name        = pr_cluster_name
```
Comment (JorTurFer) [inline terraform/main.tf:60]:
```suggestion
  cluster_name        = local.pr_cluster_name
```

File: terraform/main.tf [hunk 2]
```diff
@@ -78,7 +78,7 @@ module "azure_aks_nightly" {
   source              = "./modules/azure/aks"
   resource_group_name = var.azure_resource_group_name
   kubernetes_version  = "1.25"
-  cluster_name        = "keda-nightly-run-3"
+  cluster_name        = main_cluster_name
```
Comment (JorTurFer) [inline terraform/main.tf:81]:
```suggestion
  cluster_name        = local.main_cluster_name
```

**Review by JorTurFer — COMMENTED** [review]
In terraform, you should add the prefix `local.` when they are local variables

**Comment by JorTurFer** [comment]
LGTM!
We shouldn't merge this PR until we want to create the new infra (terraform is executed automatically on pushes to main branch). Before merging this PR we should update the secrets with the azure provider information or we will delete the current clusters and the process will end stuck due to permission gap over nodes resource group. This is because current sp isn't owner of the subscription, only the rg where we deploy the resources but nodes are in other rg created on-the-fly

**Pull request merged**